pkg/log/log.go

```diff
@@ -87,15 +87,21 @@ func MakeLogger(logger log.Logger) *FilteredLogger {
 	if verbosityFlag := flag.Lookup("v"); verbosityFlag != nil {
 		defaultVerbosity, _ = strconv.Atoi(verbosityFlag.Value.String())
 	} else {
-		defaultVerbosity = 0
+		// "the practical default level is V(2)"
+		// see https://github.com/kubernetes/community/blob/master/contributors/devel/logging.md
+		defaultVerbosity = 2
 	}
+
+	// This verbosity will be used for info logs without setting a custom verbosity level
+	defaultCurrentVerbosity := 2
+
 	return &FilteredLogger{
 		logContext:            log.NewContext(logger),
 		component:             defaultComponent,
 		filterLevel:           defaultLogLevel,
 		currentLogLevel:       defaultLogLevel,
 		verbosityLevel:        defaultVerbosity,
-		currentVerbosityLevel: defaultVerbosity,
+		currentVerbosityLevel: defaultCurrentVerbosity,
```

Comment:
Why can't we use `defaultVerbosity` here too?

Reply:
Because then all info logs (without any call to `V()`) would always be logged, because `verbosityLevel == currentVerbosityLevel`. But when I set `verbosityLevel` to `0` or `1` with the verbosity flag, I probably don't want to see all of them, only those who explicitly use `V(0)` / `V(1)`.